### DISCUSSION TITLE
once in while numbers go crazy

### DISCUSSION THREAD

**Discussion by DarkosGahan** [jomjol/AI-on-the-edge-device#2638] [General]
### The Problem

Most of the time the numbers are well recognized except maybe once every two or three days both day and night numbers get crazy. my day counter is at around 65458.4 and night 59417.8. both get crazy numbers at the same time going to 111110

### Version

15.3.0

### Logfile

```shell
[7d07h50m46s] 2023-09-27T21:20:52	<ERR>	[POSTPROC] Night: Raw: 059415.2, Value: , Status: Neg. Rate - Read:  - Raw: 059415.2 - Pre: 111110.1
```

### Expected Behavior

To not get such a big difference in numbers compared to the previous value.

**Comment by caco3** [comment]
please provide a screenshot of your device.

**Reply by DarkosGahan** [reply]
Hi,

I cannot sent a picture of the device since it is enclosed into a box to limit environmental parasites (light from the sun). but here are the details

Board : ESP32-CAM  (ESP32-S) [(ESP32-S)](https://nl.aliexpress.com/item/1005005678969617.html?spm=a2g0o.productlist.main.3.8aeb6ce268BYiG&algo_pvid=c0f447f1-98cd-4557-a47d-88881ba01c41)
FLASH : 6 x WS2812B 


Hope this helps

**Reply by caco3** [reply]
Well, I did not ask you for a **picture**, but a **screenshot**! Just open the web UI and send us the shown image in the overview.

Example: 
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/d58245c9-5070-4daa-b042-35aefdb4384c)

**Reply by DarkosGahan** [reply]
There you go

![grid_counter](https://github.com/jomjol/AI-on-the-edge-device/assets/38395649/a1fa07cd-3f75-410b-817b-1bae95772003)

**Reply by caco3** [reply]
I think your issue is the ref. marks. Try to put them farther away of each other. Eg. move the bottom one to the left side.

You could also put the camera closer.

Also I think you could make the ROIs less high.

**Reply by DarkosGahan** [reply]
thanks @caco3 I will try your recommendations.  that said, I can't put the camera closer since the grid counter is enclosed in a box with a plastic glass :( that's why the camera is actually 2 cm far from the glass and more or less 15 cm from the counter. I know it's not ideal but it's the best picture I could achieve. 

feel free to close this issue.
